index.html

```diff
@@ -1,313 +1,351 @@
+        </div>
+        <div class="work-cards">
+          <div id="w1" class="work1">
+            <div class="work1Discription" id="w1work1Discription">
+              <h2><span>Profesional Art Printing Data</span></h2>
               <p>
-                A daily selection of privately personalized reads; no accounts or
-                sign-ups required. has been the industry's standard
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
               </p>
               <ul class="category">
-                <li class="language1">html</li>
-                <li class="language2">bootstrap</li>
-                <li class="language3">Ruby</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
               </ul>
-              <button type="button">See Project</button>
             </div>
-          
-            <div class="work1" id="w2">
-              <h1>Profesional Art Printing Data</h1>
+            <button class="button" id="w1button">See Project</button>
+          </div>
+
+          <div id="w2" class="work1">
+            <div class="work1Discription" id="w2workDiscription">
+              <h2>
+                <span id="w2b">Profesional Art Printing Data</span>
+                <span id="w2d">Data Dashboard Healthcare</span>
+              </h2>
               <p>
-                A daily selection of privately personalized reads; no accounts or
-                sign-ups required. has been the industry's standard
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
               </p>
               <ul class="category">
-                <li class="language1">html</li>
-                <li class="language2">bootstrap</li>
-                <li class="language3">Ruby</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
               </ul>
-              <button type="button">See Project</button>
             </div>
-          
-            <div class="work1" id="w3">
-              <h1>Profesional Art Printing Data</h1>
+            <button class="button" id="w2button">See Project</button>
+          </div>
+
+          <div id="w3" class="work1">
+            <div class="work1Discription" id="w3work1Discription">
+              <h2>
+                <span id="w3b">Profesional Art Printing Data</span>
+                <span id="w3d">Website Protfolio </span>
+              </h2>
               <p>
-                A daily selection of privately personalized reads; no accounts or
-                sign-ups required. has been the industry's standard
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
```

Comment:
- [ ] there's a small issue with the second card in your `work-cards` section, all the cards are working fine when I hover, but only the second one is not hiding the content when I hover, kindly fix that

![image](https://user-images.githubusercontent.com/89282221/194702645-4cbeca15-fafe-4c32-8c99-c235c04d8743.png)